test/cpp/tensorexpr/test_block.cpp

```diff
@@ -0,0 +1,46 @@
+#include <sstream>
+#include <stdexcept>
+#include "test/cpp/tensorexpr/test_base.h"
+
+#include <cmath>
+
+#include "test/cpp/tensorexpr/padded_buffer.h"
+#include "torch/csrc/jit/tensorexpr/block_codegen.h"
+#include "torch/csrc/jit/tensorexpr/buffer.h"
+#include "torch/csrc/jit/tensorexpr/loopnest.h"
+#include "torch/csrc/jit/tensorexpr/tensor.h"
+
+#include <c10/util/Half.h>
+
+namespace torch {
+namespace jit {
+using namespace torch::jit::tensorexpr;
+
+static void testBlockTestVectorAdd_impl(int N, int block_size) {
+  KernelScope kernel_scope;
+  Buffer a_buf("a", kByte, {N});
+  Buffer b_buf("b", kByte, {N});
+  Tensor* c = Compute(
+      "c",
+      {
+          {N, "N"},
+      },
+      [&](const VarHandle& n) { return a_buf(n) + b_buf(n); });
+  LoopNest l({c});
+  For* n_outer;
+  For* n_inner;
+  std::vector<For*> loops = l.getLoopStmtsFor(c);
+  l.splitWithMask(loops[0], block_size, &n_outer, &n_inner);
+  l.setGPUBlockIndex(n_outer, 0);
+  l.setGPUThreadIndex(n_inner, 0);
+  //l.prepareForCodegen();
+  Stmt* stmt = l.root_stmt();
+  BlockCodeGen cuda_cg(stmt, c, a_buf, b_buf);
```

Comment:
Change the name "cuda_cg"

Reply:
I removed this outdated unit test for now. I may add a new one later.